apps/pie-docs/src/components/radio/code/code.md

```diff
@@ -0,0 +1,138 @@
+    <img alt="GitHub Workflow Status" src="https://img.shields.io/npm/v/@justeattakeaway/pie-radio.svg?label=pie-radio">
+  </a>
+
+  <a href="https://www.npmjs.com/package/@justeattakeaway/pie-webc">
+    <img alt="GitHub Workflow Status" src="https://img.shields.io/npm/v/@justeattakeaway/pie-webc.svg?label=pie-webc">
+  </a>
+</p>
+
+`pie-radio` is a Web Component built using [Lit](https://lit.dev/). It offers a simple and accessible radio button component for web applications.
+
+This component can be easily integrated into various frontend frameworks and customised through a set of properties.
+
+{% notification {
+  type: "information",
+  iconName: "engineers",
+  message: "You can try out this component on our [Storybook](https://webc.pie.design/?path=/story/radio) instance!"
+} %}
+
+## Installation
+
+To install `pie-radio` in your application, run the following on your command line:
+
+```shell
+npm i @justeattakeaway/pie-webc
+```
+
+```shell
+yarn add @justeattakeaway/pie-webc
+```
+
+{% notification {
+  type: "neutral",
+  iconName: "link",
+  message: "For more information on using PIE components as part of an application, check out the [Getting Started Guide](https://github.com/justeattakeaway/pie/wiki/Getting-started-with-PIE-Web-Components)."
+} %}
+
+## Props
+
+{% componentDetailsTable {
+  tableData: props
+} %}
+
+## Slots
+
+{% componentDetailsTable {
+  tableData: slots
+} %}
+
+## Events
+
+{% componentDetailsTable {
+  tableData: events
+} %}
+
+## Examples
+
+{% notification {
+  type: "information",
+  iconName: "engineers",
+  message: "When using multiple radio buttons, you will want to use the [Radio Group component](/components/radio-group/)"
+} %}
+
+
+For HTML:
+
+```js
+// import as module into a js file e.g. main.js
+import '@justeattakeaway/pie-webc/components/radio.js'
+```
+
+```html
+<!-- pass js file into <script> tag -->
+<pie-radio></pie-radio>
+<script>
+    const pieRadio = document.querySelector('pie-radio');
+    // Properties can be set like any JS object
+    pieRadio.checked = checked;
+</script>
+```
+
+For Native JS Applications, Vue, Angular, Svelte etc.:
+
+```js
+// Vue templates (using Nuxt 3)
+import '@justeattakeaway/pie-webc/components/radio.js';
+
+// Setup code omitted
+<pie-radio :checked="checked" value="value" @change="handleRadioChange"></pie-radio>
+```
+
+For React Applications. When using the React version of the component, please make sure you also include React as a dependency in your project as well. See Peer Dependencies section.
+
+```jsx
+import { PieRadio } from '@justeattakeaway/pie-webc/react/radio.js';
+
+// Setup code omitted
+<PieRadio
+    checked={isRadioChecked}
+    value="value"
+    onInput={handleRadioInput}>
+    {`checked: ${isRadioChecked}`}
+</PieRadio>
+```
+For NextJS 13, please directly import from the individual component package. All other template usage is the same as above.
```

Comment:
does this apply to other components too 👀 if so I think we need a ticket to update the other nextjs examples

Reply:
Good spot - as agreed offline I will remove this from all the components in a separate follow up PR 🤝